.github/workflows/build-test.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           - test-set: gen-java
             scala-version: 2.12
           - test-set: scala3
-            scala-version: 2.13
+            scala-version: 3.3
```

Comment:
This is what I missed in reference to https://github.com/apache/incubator-pekko-grpc/pull/66#issuecomment-1565661703